spec/suites/SpecHelper.js

```diff
@@ -79,5 +79,33 @@ var touchEventType = L.Browser.touchNative ? 'touch' : 'pointer'; // eslint-disa
 // Note: this override is needed to workaround prosthetic-hand fail,
 //       see https://github.com/Leaflet/prosthetic-hand/issues/14
 
+function createContainer() {
+	var container = document.createElement("div");
+	container.style.position = 'absolute';
+	container.style.top = '0px';
+	container.style.left = '0px';
+	container.style.height = '400px';
+	container.style.width = '400px';
+	container.style.opacity = '0.4';
+	document.body.appendChild(container);
+
+	return container;
+}
+
+function createMapContainer(mapOptions) { /* eslint-disable-line no-unused-vars */
+	var container = createContainer();
+	var map = L.map(container, mapOptions);
+	return {map: map, container: container};
+}

```

Comment:
I find this particular function redundant — you return an object, then unroll it:

```js
var obj = createMapContainer(options);
map = obj.map;
container = obj.container;
```

An equivalent is both more explicit and simpler:

```js
container = createContainer();
map = L.map(container, options);
```

Reply:
With keeping the map in an own option is makes it possible to adjust while developing only one place. For example when I want to implement touchevents and I debug failed tests, it can be very useful if there is only one function which creates the map and I need only to edit one place to add for example listeners to the map.
I think if we always do the same (like creating the map instance) we can do this i one central place.

Would it be better if I move the map creation into a own function without creating the container?

Reply:
I would introduce new utility functions when they are actually needed, instead of trying to anticipate a chance of something being useful in the future. The more explicit the code, the better.

Reply:
It was a feature that I needed to debug else I would suggest it 😄 but is ok for me, I will replace it